client/src/routes.js

```diff
@@ -62,6 +65,18 @@ const routes = [
     path: "/about",
     component: About,
   },
+  {
+    path: "/CreateOrganizationProfile",
```

Comment:
Let's change this to be `/create-organization-profile` to stay consistent

Reply:
This will be fixed by #399

Reply:
Im sorry but you re reviewing the wrong page. This PR is for edit-organization-profile and edit-organization-account only @anniegiang